cloud-chatops/src/dev.py

```diff
@@ -0,0 +1,71 @@
+"""This module runs local integration tests of the code."""
+
+import logging
+import argparse
+from argparse import Namespace
+from errors import NoTestCase
+from read_data import validate_required_files
+from events import run_global_reminder, run_personal_reminder
+
+logging.basicConfig(level=logging.DEBUG)
+
+# Disable this warning as the variable is not constant.
+# Empty variable here to patch when testing.
+# pylint: disable=C0103
+args = None
+
+
+def run_tests() -> None:
```

Comment:
Nit: Run test makes me think of unit testing
I think it should be called something like `run_methods` or similar since we know we're in `dev.py` which implies dev testing